bond_app/datastore_cache_api.py

```diff
@@ -29,11 +29,15 @@ def add(self, key, value, expires_in=0, namespace=None):
                    expires_at=DatastoreCacheApi._calculate_expiration(expires_in)).put()
         return True
 
-    def get(self, key, namespace=None):
+    def get_entry(self, key, namespace=None):
         entry = DatastoreCacheApi._build_cache_key(key, namespace).get()
         if not entry or entry.expires_at < datetime.datetime.now():
             return None
-        return entry.value
+        return entry
```

Comment:
I went back and forth on whether or not I should be sending CacheEntry objects to other parts of the code, but these modules are on the same "level", so it seemed ok, and it means we don't have to package up this data in the cache in some other way (a dict, a tuple, etc.)